meshio/_plot3d.py

```diff
@@ -0,0 +1,94 @@
+"""
+PLOT3D format is for structured grids only
+For more info, check out: PLOT3D User's Manual, chapter 8: Data File Formats
+Author: Samir OUCHENE
+"""
```

Comment:
Please put a link to the PLOT3D format specification.

Reply:
The wikipedia article does not contain much information. However, the format specification can be found in the user's manual: https://ntrs.nasa.gov/archive/nasa/casi.ntrs.nasa.gov/19900013774_1990013774.pdf

Reply:
We'll need the link in the file, not here.